src/components/ItemPopover.vue

```diff
@@ -8,6 +8,13 @@
   >
     <div class="popup d-flex flex-column align-items-center">
       <h6 class="title">{{item.name}}</h6>
+      <p v-if="item.equipmentSlot" class="equipment-slot-name text-uppercase">{{item.equipmentSlot}}</p>
+      <equipment-slot

```

Comment:
Last thing: when these popovers are too tall, it can really screw up mobile users from being able to sell things out of their inventory.

I might recommend hiding this on mobile with a `class="d-none d-md-block"`

Reply:
I realise now that the PR was incomplete in regards to the import.
But since I removed EquipmentSlot (makes sense of course) it's no longer needed.
It just displays the img directly as per your suggestion.

In regards to mobile users, I'm not sure I would recommend showing them any less information.
I made the icon a bit smaller om mobile screens, and the new line should be close to the height of the "Equippable" status-line which now is removed.

But I'll change it if you want to! Take a look and give me a heads up ;)